buildconfig/CMake/FindBoost.cmake

```diff
@@ -1,3 +1,6 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
If we upgrade cmake as discussed can we remove this file?

Reply:
Won't it still be needed for other platforms? Or is this strictly used for windows?

Reply:
Good question. I misunderstood and thought we added it here. I'm not sure why we had it in the first place could you ask on Slack if anyone knows?